### DISCUSSION TITLE
Clarification on  "General Contract Provisions"

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#42] [Q&A] [answered]
There was recently new examples for stop loss provisions added and based on the location and example text, what it appears is that you are now wanting us to indicate stop loss clauses for ALL contract profiles under this one field. So when I have 50 plan names and 30 of them have a stop loss clause, there will be 30 different statements in this field, is that the intent? 

Just to fully understand, if we take a DRG for example, we have a negotiated dollar,  we also need to add to the algorithm field that this could be adjusted for outliers, and rather than adding that outlier language to the additional payer notes, we need to combine them all into one?       

In my opinion, since this qualifies as an algorithm, rather than stating that this could be adjusted for outliers in that field, this should be where the stop loss information goes.  

But please provide clarification

**Comment by RidenourK** [comment, marked as answer]
@JaSimps5. Thank you for your question.

As indicated in the CSV and JSON documentation, the “general_contract_provisions” data element is an optional data element that can be used to encode payer contract provisions that are applicable at an aggregate level and may include variable items and services. Examples could be "stop-loss" provisions or "lesser than" provisions that apply to the claim (as opposed to each item or service on the claim). Multiple general contract provisions across payer and plan combinations can be included in this data element. If a contract provision applies to a specific item or service, use the Payer-specific Negotiated Charge: Algorithm data element and encode the Estimated Allowed Amount data element.
